linux-packages/packaging-scripts/create-linux-packages.sh

```diff
@@ -129,14 +129,18 @@ fpm --version "$VERSION" \
   --vendor 'Octopus Deploy' \
   --url 'https://octopus.com/' \
   --description "$PACKAGE_DESC" \
-  --rpm-sign \
   --verbose \
   "${FPM_RPM_OPTS[@]}" \
   "${FPM_OPTS[@]}" \
   "$INPUT_PATH/=$INSTALL_PATH/" \
   tmp_usr_bin/=/usr/bin/
 set +ex
 
+echo "Signing .rpm package."
+set -ex
+rpmsign --addsign *.rpm
+set +ex
```

Comment:
Interesting that the other way stopped working

Reply:
Yeah, Debian version was updated, and rpm no longer supports the --sign option